### DISCUSSION TITLE
Unable to install slepc

### DISCUSSION THREAD

**Discussion by huisophiewang** [firedrakeproject/firedrake#3515] [Firedrake support] [answered]
I was able to install firedrake without any error. But the demo code I need to run requires slepc, and I had this error "Unable to import SLEPc, eigenvalue computationnot possible (try firedrake-update --slepc)." 

I have tried "firedrake-update --slepc" and "firedrake-install --slepc", but neither works. Here are the log files. I really appreciate any help with it. 

firedrake:
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/15052853/firedrake-install.log)
petsc:
[configure.log](https://github.com/firedrakeproject/firedrake/files/15052856/configure.log)
slepc:
[configure.log](https://github.com/firedrakeproject/firedrake/files/15052863/configure.log)
[make.log](https://github.com/firedrakeproject/firedrake/files/15052874/make.log)

**Comment by ksagiyam** [comment, marked as answer]
`gethostbyname failed`

The workaround mentioned here https://www.firedrakeproject.org/download.html#testing-the-installation probably helps.

**Reply by huisophiewang** [reply]
Yes, the local host update works. I'm finally able to run the code without any problem. Thank you so much!
